ktor-network/ktor-network-tls/jvm/src/io/ktor/network/tls/TLSClientSessionJvm.kt

```diff
@@ -49,15 +60,15 @@ private class TLSSocket(
 
     private suspend fun appDataInputLoop(pipe: ByteWriteChannel) {
         try {
-            input.consumeEach { record ->
+            base.input.consumeEach { record ->
                 val packet = record.packet
                 val length = packet.remaining
                 when (record.type) {
                     TLSRecordType.ApplicationData -> {
                         pipe.writePacket(record.packet)
                         pipe.flush()
                     }
-                    else -> throw TLSException("Unexpected record ${record.type} ($length bytes)")
+                    else -> throw TlsException("Unexpected record ${record.type} ($length bytes)")
```

Comment:
We have to throw `TLSException` for compatibility with code which hasn't been updated to use the new type in catch 😢